src/templates/Meetup/index.jsx

```diff
@@ -215,6 +218,16 @@ export default ({
             </a>
           </SubmitTalkButton>
         </section>
+        {allAirtable.totalCount > 0 ? (
+          <section>
+            <h2><span role="img" aria-label="busts in silhouette">👥</span> Attendees <span role="img" aria-label="busts in silhouetee">👥</span></h2>
+            <p>{allAirtable.totalCount} attendees</p>
+            {/* @TODO: Put this back when we have added the ability for people to hide their profile */}
```

Comment:
I think for now we can still display this but put a caption to ask people to put down "react-knowledgeable" in their GitHub handle if they do not want to display their attendance. This way we still get the RSVP but we show our own avatar.

Reply:
Actually, before we implement the Github OAuth, people can just leave the username field empty. Then we won't know it and hence won't show their avatar.

Reply:
hmm, I am slightly leaning towards having GitHub OAuth for sign in land together with this PR then.